.kitchen.yml

```diff
@@ -25,6 +28,23 @@ suites:
                 port: 6379,
               }
             ]
+  - name: redis-package
```

Comment:
There is no matching spec file for this suite.  It ends up resulting in a converge but no actual verification that redis is installed via package method and working properly.  Do you feel comfortable adding a matching spec file for this with some basic tests?  If not I can add one post merge.